Makefile

```diff
@@ -1,4 +1,4 @@
-RUN_SETTINGS?=staging
+RUN_SETTINGS?=nightlies
```

Comment:
It is an expected changes? Anyway I think it make sense to default to nightlies.

Reply:
Yeah, I figured everyone that tries to debug something is going to wanting "nightlies" rather than "staging" which becomes stale.